### ISSUE TITLE
Shortcuts/Features are not working

### ISSUE THREAD

**Issue by A-n-d-y-K** [kensam94/joplin-plugin-eztable#1]
### Description
Some of listed features are working only via menu. Shortcuts are not working.
I´m not sure if it is a tools / version incompability issue. Therfore I list below the versions on my machine.
Thanks in advance for your support

### Listed features
-    Insert empty table (Ctrl + Shift + I) -> Working
-    Format Table (Ctrl + Shift + F) -> Not Working
-    Insert new row (Ctrl + Enter) -> Not Working
-    Insert new column (Ctrl + Tab) -> Not Working
-    Delete column (Access by right click) -> Working
-    Hit enter to insert \<br> as a newline in the table- > Working
-    Hit tab to move cursor to the next column -> working

### Tool versions
- Joplin for Desktop 2.8.8
- EzTable v1.0.2

### Info
- Editor monospace font family set to "Cascadia Mono"
- Format Table - via Menu: Tools -> Ez Table -> Working
- Insert new row - via Menu: Tools -> Ez Table -> Working
-  Insert new column - via Menu: Tools -> Ez Table -> Working

**Comment by kensam94** [comment]
@A-n-d-y-K Is your cursor located at the same place for your working and not working case? Is there any other shortcut conflict with these?

**Comment by A-n-d-y-K** [comment]
@kensam94 ,
thanks for your response, I played arrouned with curser position. Before, Within, after Table. I found no differnece to what I worte in Issue description.
Best regards

**Comment by xXZaretXx** [comment]
@kensam94  Hello kensam,
i have the same problem. I found out, that i can set :
 Ez Delete Column, Ez Format Table and Ez Insert Table in the Keyboard Shortcuts Option from Joplin.  
After setting the shortcut, these 3 commands work.

**Comment by dsouzadrian** [comment]
Yup, same issue here, I was able to set the keyboard shortcuts for delete, format and insert as @xXZaretXx  mentioned and got those working.

The other shortcuts for insert row and insert new column thought do not show up under Joplin keybaord shortcut options.

**Comment by dsouzadrian** [comment]
Not sure what I did, but I was playing around with some files in the profile to see if I could manually insert those keyboard shortcuts. I tried adding one for insert row and restarted Joplin, after restarting all keyboard shortcuts showed up as expected.

![image](https://user-images.githubusercontent.com/8682704/216063660-e9ff537c-d353-4e62-be6b-de13d7e2e9df.png)

Removed whatever mods I'd made and the keyboard shortcuts still work as expected.

**Comment by backtrackshubham** [comment]
Hi @dsouzadrian could you please share keymap-desktop.json file, insight is much appriciated

**Comment by XanatosX** [comment]
Same issue for me, seems like nothing was set

![grafik](https://github.com/kensam94/joplin-plugin-eztable/assets/10531466/25c41afb-f6a5-446b-b18e-3ef50feeebb2)

And looks like the following are missing

- Insert New Row
- Insert New Column

Joplin Version: 2.11.11 (Win32)
Plugin Version: 1.0.2

Other Plugins installed

![grafik](https://github.com/kensam94/joplin-plugin-eztable/assets/10531466/03592c22-0f46-4074-834f-8191d9f98d8b)

**Comment by XanatosX** [comment]
I replaced the plugin with the following https://github.com/FelisDiligens/joplin-plugin-multimd-table-tools seems like this works better for me.

**Comment by uyrussell21** [comment]
Had to use this plugin since [the other table markdown plugin](https://github.com/FelisDiligens/joplin-plugin-multimd-table-tools) was crashing on some instances.

Keyboard shortcuts were not set until I installed both plugins, weird behavior but here's the keymap json (windows):

```json
[
  {
    "command": "ezInsertTable",
    "accelerator": "Ctrl+Shift+I"
  },
  {
    "command": "ezFormatTable",
    "accelerator": "Ctrl+Shift+F"
  },
  {
    "command": "ezDeleteCol",
    "accelerator": null
  },
  {
    "command": "ezInsertNewRow",
    "accelerator": "Ctrl+Enter"
  },
  {
    "command": "ezInsertNewCol",
    "accelerator": "Ctrl+Tab"
  }
]
```
